frontend/package.json

```diff
@@ -22,11 +22,13 @@
     "discord-webhook-node": "^1.1.8",
     "framer-motion": "^6",
     "lodash": "^4.17.21",
-    "next": "13.0.2",
+    "next": "^13.5.5",
+    "node": "20.10.0",
```

Comment:
this is downloading the NPM package [node](https://www.npmjs.com/package/node) and installing as a dependency. we should instead change the engine, rather than making node a dependency.